README.md

```
@@ -1,101 +1,233 @@
+  ],
+  cache: [
+    default_ttl: :timer.minutes(5),
+    cleanup_interval: :timer.minutes(10)
+  ]
+```
+
+## Monitoring
+
+The service provides comprehensive monitoring with 5-minute status reports:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)